Pipfile

```diff
@@ -0,0 +1,8 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+
+[packages]
+python-snappy = "*"
```

Comment:
Should this apply to requirements.txt as well with minimum versions instead of exact (==)?